README.md

```diff
@@ -130,13 +130,33 @@ developers monitor both channels.
 
 ### News and Updates 📣
 
+- **Feb 2024**: Version 0.14 is released, which deprecates [jj checkout and merge](changelog.md#deprecations),
+  and `jj init --git`, which is now just called `jj git init`. 
```

Comment:
nit: maybe ", as well as `jj init --git`, which ..." to try to make it clearer that the `jj init --git` bit is distinct from the `jj checkout/merge`

Reply:
done